recirq/quantum_chess/caching_utils.py

```diff
@@ -11,3 +12,11 @@ class CacheKey:
 
 def cache_key_from_move(m: move.Move, repetitions: int) -> CacheKey:
     return CacheKey(m.move_type, repetitions)
+
+
+@dataclass(frozen=True)
+class ProbabilityHistory:
```

Comment:
Add doc string with Params rather than line comments.

Reply:
I'm not familiar with a Params section, but I put the stuff in a doc string.

Reply:
I think the docstring below is supposed to be Args, not Attributes and have colons, not dashes:

```
Dataclass for storing histograms of board position probabilities from a given point in the move history.

Args:
    repetitions: number of samples used to generate probabilities
    probabilities: maps square -> probability of being occupied
    full_squares: (derived from self.probabilities) full squares bitboard
    empty_squares: (derived from self.probabilities) empty squares bitboard
```

Reply:
Done.

Reply:
Indentation is still not correct, and you are using dashes instead of colons.

Reply:
All right. Is there some style guide for this repo that I should be consulting?

Reply:
Here's the Google python style guide:
https://google.github.io/styleguide/pyguide.html#doc-function-args

The docstring formatting is so that any docs can be parsed and show up correctly on the documentation, though I don't think the reCirq stuff is currently listed on quantumai.google.